.github/workflows/build-image.yml

```
@@ -1,19 +1,40 @@
-name: Build Image from Dockerfile
+name: Build Image from Dockerfile 
 on:
   push:
     branches: [ master ]
 
 jobs:
-  build:
-    name: Build image
+  build_x86_wheels:
+    name: Build image for x86
```

```suggestion
  build_x86_64_wheels:
    name: Build image for x86_64
```

x86 is generally used to mean the 32-bit architecture, which we're not doing at the moment.